WalletWasabi.Fluent/ViewModels/Wallets/CoinJoinStateViewModel.cs

```diff
@@ -93,7 +93,7 @@ public CoinJoinStateViewModel(WalletViewModel walletVm)
 		{
 			if (!wallet.KeyManager.IsCoinjoinProfileSelected)
 			{
-				await RoutableViewModel.NavigateDialogAsync(new CoinJoinProfilesViewModel(wallet.KeyManager, isNewWallet: false), NavigationTarget.DialogScreen);
+				await UiContext.Navigate().NavigateDialogAsync(new CoinJoinProfilesViewModel(wallet.KeyManager, isNewWallet: false), NavigationTarget.DialogScreen);
```

Comment:
same here, pass navigation target to `Navigate()` instead of `NavigateDialogAsync()`?